app/src/main/java/net/squanchy/schedule/view/SpeakerView.java

```diff
@@ -25,6 +35,7 @@ public SpeakerView(Context context, AttributeSet attrs, int defStyleAttr) {
     public SpeakerView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
         super(context, attrs, defStyleAttr, defStyleRes);
 
+        imageLoader = ImageLoaderInjector.obtain(context).imageLoader();
```

Comment:
Seems reasonable given there's going to be a handful of those only (only that many cards you can fit on the screen, and it's in a recyclerview, so there's going to be less than ten). Thought it'd be ok as first draft. Up for improving tho.